CONTRIBUTING.md

```diff
@@ -19,29 +19,25 @@ glide install
 go build github.com/mysterium/node
```

Comment:
This is not valid also

Reply:
Step changed from "Compile code" to "Fetch dependencies"